packages/create-app/src/prompts/promptTemplate.ts

```diff
@@ -35,7 +35,9 @@ export async function promptTemplate(
         : "Select a framework:",
       {
         type: "select",
-        options: TEMPLATES.map((template) => ({
+        options: TEMPLATES.filter(template =>
+          template.id !== "template-tutorial-todo-app" // Hide tutorial template
```

Comment:
suggestion: add `hidden?: boolean` to the `Template` interface and check `!template.hidden` here